MentoringProject.Tests/UserServiceTest.cs

```diff
@@ -0,0 +1,189 @@
+using Mentoring_project.Entities;
+using Mentoring_project.Interfaces;
+using Mentoring_project.Repositories;
+using Mentoring_project.Services;
+using MentoringProject.Tests;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Mentoring_project.Test
+{
+    public class UserServiceTest : IClassFixture<DbFixture>
+    {
+        private DbFixture _fixture;
+        public UserServiceTest(DbFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void GetUserByIdTest()
+        {
+            using (var context = new DbProjectContext(_fixture.GetDbOptions()))
+            {
+                //Arrange
+                var uow = new UnitOfWork(context);
+                var userService = new UserService(uow);
+                var expectedUser = context.Users.First();
+
+                // Act
+                var result = userService.GetUserById(expectedUser.UserId);
+
+                //Assert
+                Assert.Equal(expectedUser, result);
```

Comment:
Use FluentAssertion to compare objects 
result.Should().Be().[Same/Equel](expectedUser)